src/tools/antool/listsyscalls.py

```diff
@@ -881,6 +883,12 @@ def add_to_unsupported_lists(self, syscall, name, l_names, l_inds, relative):
 
     ####################################################################################################################
     def add_to_unsupported_lists_or_print(self, syscall):
+        if syscall.unsupported == RESULT_SUPPORTED:
```

Comment:
This reads really weird... It should be just bool with no negation in the name...

Reply:
There are four types of unsupported syscalls:
```
RESULT_UNSUPPORTED_YET = 1
RESULT_UNSUPPORTED_RELATIVE = 2
RESULT_UNSUPPORTED_FLAG = 3
RESULT_UNSUPPORTED = 4
```